src/vs/code/electron-browser/sharedProcess/sharedProcessMain.ts

```diff
@@ -294,6 +300,13 @@ class SharedProcessMain extends Disposable {
 		// Extension Host
 		services.set(IExtensionHostStarter, this._register(new ExtensionHostStarter(logService)));
 
+		// Signing
+		services.set(ISignService, new SyncDescriptor(SignService));
+
+		// Tunnel
+		services.set(ITunnelService, new SyncDescriptor(TunnelService));
```

Comment:
Is `TunnelService` still needed in src\vs\code\electron-main\main.ts now?

Reply:
I haven't touched any of the usages in `/electron-main/`. I think perhaps the webview needs that in the main process.